modal/_runtime/container_io_manager.py

```diff
@@ -278,11 +278,13 @@ async def call_generator_async(self) -> AsyncGenerator[Any, None]:
         logger.debug(f"Finished generator input {self.input_ids}")
 
     async def output_items_cancellation(self, started_at: float):
+        output_created_at = time.time()
         # Create terminated outputs for these inputs to signal that the cancellations have been completed.
         return [
             api_pb2.FunctionPutOutputsItem(
                 input_id=input_id,
                 input_started_at=started_at,
+                output_created_at=output_created_at,
```

Comment:
Do you see a good/quick place to test this somewhere in https://github.com/modal-labs/modal-client/blob/main/test/container_app_test.py ?